webui/react/src/App.tsx

```diff
@@ -63,6 +64,7 @@ const AppView: React.FC = () => {
       const message = 'New WebUI Version';
       const description = <div>
         WebUI version <b>v{info.version}</b> is available.
+        Read <Link external path="/docs/release-notes.html">release notes</Link> in our docs.
```

Comment:
@ahj-determined any thoughts on how should we word this?

Reply:
@hamidzr Maybe something like: See what's new in our <Link external path="/docs/release-notes.html">release notes</Link>